servers/src/grin/seed.rs

```diff
@@ -186,7 +186,7 @@ fn monitor_peers(
 	// maintenance step first, clean up p2p server peers
 	peers.clean_peers(config.peer_max_count() as usize);
 
-	if peers.healthy_peers_mix() {
+	if peers.peer_count() > peers.peer_outbound_count() && peers.healthy_peers_mix() {
```

Comment:
Should that be `>=`?

Reply:
Don’t think so, but not sure anymore:) the logic is - exit if we have at least 1 inbound peer and we have enough outbounds

Reply:
But for NAT'd peers there'll never be any inbound? So they could have 100 oubounds and this will still never return true?

Reply:
Oh, wrong place to add this check, thanks for spotting it. It's supposed to be where we connect